.coveragerc

```diff
@@ -1,3 +1,6 @@
+[report]
+precision = 2
```

Comment:
Enabling the local code coverage to report the partial percentage so one can see the same as the codecov reports.